README.md

```diff
@@ -1,4 +1,4 @@
-Xamarin Community Toolkit
+<img src="assets/XamarinCommunityToolkit_128x128.png" width="64" /> Xamarin Community Toolkit
```

Comment:
BTW does it work with a relative url path?

Reply:
I see, it works 
![image](https://user-images.githubusercontent.com/10124814/92790266-4bd2f200-f3b4-11ea-8f30-c9573a76a13c.png)

Reply:
One way to find out... YOLO :D